web/src/games/bombsAndBunnies/components/BetDisplay.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import css from './BetDisplay.css';
+
+export interface IBetDisplayProps {
+  currentBet: number;
+  revealedCount?: number;
+  isRevealing: boolean;
+}
+
+export class BetDisplay extends React.Component<IBetDisplayProps> {
+  text: string = '';
+  activeText: string = '';
+  flip: boolean = false;
+
+  render() {
+    var text;
```

Comment:
please use `let` instead of `var` ... `var`s are global, `let`s are scoped.

Reply:
or `const` if the variable isnt changed

Reply:
I'll have a scan through and change them where they make sense